From 20ed7b24df05eadf83168d1d0ce0052a31380928 Mon Sep 17 00:00:00 2001
From: Stephen Hemminger <shemminger@vyatta.com>
Date: Wed, 15 Feb 2012 10:05:45 -0800
Subject: [PATCH] dhcp-client-script: don't use /tmp

/tmp is a dangerous place and better to put log files in /var/log.
Based on patch by Vasiliy Kulikov <segoon@openwall.com>
---
 examples/dhcp-client-script |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/examples/dhcp-client-script b/examples/dhcp-client-script
index 7207b57..f39bc10 100644
--- a/examples/dhcp-client-script
+++ b/examples/dhcp-client-script
@@ -14,7 +14,7 @@
 # we should install and preserve.
 #
 
-exec >> /tmp/DHS.log 2>&1
+exec >> /var/log/DHS.log 2>&1
 
 echo dhc-script $* reason=$reason
 set | grep "^\(old_\|new_\|check_\)"
-- 
1.7.6.5

